pynecone/utils.py

```diff
@@ -532,30 +532,38 @@ def get_num_workers() -> int:
     return (os.cpu_count() or 1) * 2 + 1
 
 
-def run_backend(app_name: str):
+def run_backend(app_name: str, loglevel: str = "error"):
     """Run the backend.
 
     Args:
         app_name: The app name.
+        loglevel: The log level.
     """
-    command = constants.RUN_BACKEND + [
-        f"{app_name}:{constants.APP_VAR}.{constants.API_VAR}"
-    ]
-    subprocess.run(command)
+    from uvicorn import run
```

Comment:
Can we move this import to the top as `import uvicorn` and call `uvicorn.run` here